poncho/src/poncho/package_create.py

```diff
@@ -98,51 +98,74 @@ def _copy_run_in_env(env_dir):
     os.chmod(f'{env_dir}/env/bin/run_in_env', 0o755)
 
 
-def pack_env(spec, output, conda_executable=None, download_micromamba=None):
-    # record packages installed as editable from pip
-    local_pip_pkgs = _find_local_pip()
+def pack_env(spec, output, conda_executable=None, download_micromamba=None, ignore_editable_packages=None):
+    # pack a conda directory directly
+    if not os.path.isfile(spec) and spec != "-":
+        # remove trailing slash if present
+        spec = spec[:-1] if spec[-1] == '/' else spec 
+        try:
+            logger.info('packaging the environment...')
+            _copy_run_in_env(spec)
+            os.makedirs(f'{spec}/bin/', exist_ok=True)
+            os.rename(f'{spec}/env/bin/poncho_package_run', f'{spec}/bin/poncho_package_run')
```

Comment:
This function is already too long. Let's split it in your new case, and the case that was there before.